app/src/main/java/com/digitalarchitects/rmc_app/components/CommonUI.kt

```diff
@@ -832,3 +834,28 @@ fun RmcTopButton() {
     }
 }
 
+@Composable
+fun BooleanToggle(
+    value: Boolean,
+    onToggle: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    contentDescription: String? = null,
+    size: Dp = 80.dp,
+    iconSize: Dp = 80.dp
+) {
+    Box(
+        modifier = modifier
+            .size(size)
+            .clickable { onToggle(!value) }
+    ) {
+        val icon = if (value) Icons.Default.ToggleOn else Icons.Default.ToggleOff
+        val tint = if (value) MaterialTheme.colorScheme.primary else Color.Gray
```

Comment:
Voor een active switch, maar die niet ON is maar OFF, zou ik MaterialTheme.colorScheme.secondaryContainer gebruiken voor nu i.p.v. Gray

Reply:
aangepast, thanks! Wist niet zo goed welke kleur handig was.